### ISSUE TITLE
Resize of campaign overview - number in circle graphs moves with resize

### ISSUE THREAD

**Issue by krasian** [gophish/gophish#901]
What version of Gophish are you using?: 0.4

Brief description of the issue: When resizing your browser window, the number for the associated "circle" graph moves with the resize.  At times, the number may be obscured.

What are you expecting to see happen? :  Perhaps the number consistently in the center of the circle.

What are you seeing happen?  Number moves around.  Please see attached images.
![large](https://user-images.githubusercontent.com/26934540/34416106-f0b9b556-ebbf-11e7-92bf-b0d0535c360c.PNG)
![small](https://user-images.githubusercontent.com/26934540/34416109-f5f216d0-ebbf-11e7-8f8a-0001f59cff9a.PNG)

**Comment by S0larflare** [comment]
That’s odd, what browser is that in?

Sent from my iPhone

On 28 Dec 2017, at 16:12, krasian <notifications@github.com<mailto:notifications@github.com>> wrote:


What version of Gophish are you using?: 0.4

Brief description of the issue: When resizing your browser window, the number for the associated "circle" graph moves with the resize. At times, the number may be obscured.

What are you expecting to see happen? : Perhaps the number consistently in the center of the circle.

What are you seeing happen? Number moves around. Please see attached images.
[large]<https://user-images.githubusercontent.com/26934540/34416106-f0b9b556-ebbf-11e7-92bf-b0d0535c360c.PNG>
[small]<https://user-images.githubusercontent.com/26934540/34416109-f5f216d0-ebbf-11e7-8f8a-0001f59cff9a.PNG>

**Comment by krasian** [comment]
Google Chrome Version 63.0.3239.108 (Official Build) (64-bit)

**Comment by jordan-wright** [comment]
Yes, this is a known issue. This will take some work to fix, since the number placement is dynamic at the time of the chart creation. I might try adding an event listener to catch resize events and hopefully move the numbers to the correct spot.

Thanks for the report!

**Comment by AyushGupta11442** [comment]
i think i can try this issue so can i work on it ?
